index.html

```diff
@@ -17,58 +17,7 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <div class="PriceBar">
-     <div class="header"> 
-      <p class="balance">My balance</p>
-      <p class="myBalance">$921.48</p>
-      <img src="images/logo.svg" alt="logo" class="img-logo">
-        
-    </div>
-     <div class="main">
-      <p class="spending">Spending - Last 7 days
-      </p>
-      <div class="monday">
-
-      </div>
-      <p class="daytext">mon</p>
-    
-      <div class="tuesday">
-
-      </div>
-      <p class="daytext">tue</p>
-      <div class="Wednesday">
-
-      </div>
-      <p class="daytext">wed</p>
-      <div class="Thursday">
-
-      </div>
-      <p class="daytext">thu</p>
-      <div class="Friday">
-
-      </div>
-      <p class="daytext">fri</p>
-      <div class="Saturday">
-
-      </div>
-      <p class="daytext">sat</p>
-      <div class="Sundey">
-
-      </div>
-      <p class="daytext">sun</p>
-      <div class="rectangle"> </div>
-      <div>
-        <p class="total">Total this month</p>
-        <p class="expense_value">$478.33</p>
-      </div>
-      <div>
-        <p class="Pecentage_increase">+2.4% </p>
-        <p class="Prevous_month">from last month</p>
-      </div>
```

Comment:
Could you consider adding a space between each "div" to give your code a more professional documentation.